README.md

```diff
@@ -1,2 +1,202 @@
-# recipe-ror-app
-Using ROR to create a cooking app that tracks inventories, ingredients, and recipes. Users may create recipes, manage their saved information, build shopping lists based on what they have and what they need to complete a dish, and save ingredients.
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] OPTIONAL
Kindly ensure to remove the Microverse template logo. Optionally, you can replace it with your own logo if you have one.